mvt/common/indicators.py

```diff
@@ -40,15 +40,20 @@ def _load_downloaded_indicators(self) -> None:
 
     def _check_stix2_env_variable(self) -> None:
         """
-        Checks if a variable MVT_STIX2 contains path to a STIX files.
+        Checks if a variable MVT_STIX2 contains path to a STIX file. Also recursively searches through dirs in MVT_STIX2
         """
         if "MVT_STIX2" not in os.environ:
             return
 
         paths = os.environ["MVT_STIX2"].split(":")
         for path in paths:
-            if os.path.isfile(path):
+            if os.path.isfile(path) and path.lower().endswith(".stix2"):
                 self.parse_stix2(path)
+            if os.path.isdir(path):
+                for root, dirs, files in os.walk(path):
+                    for filename in files:
+                        if filename.lower().endswith(".stix2"):
```

Comment:
Non-blocker: we might have some examples of using glob that could condense this into one for loop

Reply:
happy to change that if you can show me an example.

Reply:
I think he means something like [this](https://github.com/mvt-project/mvt/blob/efe46d7b49d383d97151e03bbde3da6970f00f5f/mvt/ios/decrypt.py#L134) using the [glob library](https://docs.python.org/3/library/glob.html)

Reply:
> I think he means something like [this](https://github.com/mvt-project/mvt/blob/efe46d7b49d383d97151e03bbde3da6970f00f5f/mvt/ios/decrypt.py#L134) using the [glob library](https://docs.python.org/3/library/glob.html)

Yes exactly. The glob makes it easier to rationalise about what the code is doing IMO.

Reply:
I changed the code to use glob.glob instead. 
note: this is case-sensitive now. Case insensitivity with glob was only added in python3.12